src/components/ebay-carousel/README.md

```diff
@@ -11,8 +11,6 @@
 ```
 
 This component will bundle different resources depending on Lasso flags provided by your application.
```

Comment:
Not sure this comment is still needed, either, since all other components also handle the `skin-ds6` flag and you've removed the `touch/no-touch` flags from just this one component.